app/scripts/lib/validate.js

```diff
@@ -103,7 +103,41 @@ define([
       }
 
       return password.length >= Constants.PASSWORD_MIN_LENGTH;
+    },
+
+    /**
```

Comment:
I agree with that, I didn't find a better lib file and I wasn't confident enough to create new one.
So where should I put it? should I make  a new file? or add it to another o?